Revit_Core_Engine/Query/ConnectedElements.cs

```diff
@@ -45,24 +45,10 @@ public static List<Element> ConnectedElements(this Element element)
                 return null;
             }
             
-            ConnectorSet connSet = null;
-            if (element is FamilyInstance)
-            {
-                connSet = (element as FamilyInstance).MEPModel?.ConnectorManager?.Connectors;
-            }
-            else if (element is MEPCurve)
-            {
-                connSet = (element as MEPCurve).ConnectorManager?.Connectors;
-            }
-            
-            if (connSet == null)
-            {
-                BH.Engine.Base.Compute.RecordError("Input element is not supported by the method. Check if element is a valid MEP object.");
-                return null;
-            }
+            List<Connector> connectors = element.Connectors();
+            List<Element> connectedElements = new List<Element>();
```

Comment:
You can make the method look more neat by using `HashSet<ElementId>`

```suggestion
            HashSet<ElementId> connectedElements = new HashSet<ElementId>();
```

...then:
```C#
(...)
 if (el != null && el.Id != element.Id)
    connectedElements.Add(el.Id);
```